### PR TITLE
Loading parameterizer and registry

### PR DESCRIPTION
**Pull request by sofiatti**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "REPL for the Computable Protocol",
   "main": "index.js",
   "scripts": {
-    "test": "jasmine ./**/*.spec.js"
+    "test": "jasmine ./**/*.spec.js",
+    "start": "node server.js"
```
Comment (geoffkinder) [inline package.json:8]: Do we start the repl with `node repl.js`? Maybe we could make that a command also?

File: repl.js [hunk 2]
```diff
@@ -67,6 +67,26 @@ const setup = async () => {
     dllAddress: replServer.context.dll.options.address,
     attributeStoreAddress: replServer.context.attributeStore.options.address
   }, { gas: GAS_LIMIT, gasPrice: GAS_PRICE })
+
+  // parameterizer
+  replServer.context.dll = await deployDll(replServer.context.web3, replServer.context.admin)
+  replServer.context.attributeStore = await deployAttributeStore(replServer.context.web3, replServer.context.admin)
```
Comment (geoffkinder) [inline repl.js]: Do we need to deploy the `dll` and the `attributeStore` multiple times in this block? Not sure what happens if we do, but it looks like they are all written back to the same variable in `replServer.context`.

Comment (robrobbins) [reply repl.js]: do not deploy multiple times, it wouldn't break the context, but you'd have multiple deployed contracts on the ganache instance

**Comment by sofiatti** [comment]
Removed the extra deploy lines. Thanks, @SUICIDALcow, and @robrobbins! 

I'll make another PR later addressing the defaults for the Parameterizer after our chat today.

**Pull request merged**